docs/src/test-api/class-test.md

```diff
@@ -920,6 +920,42 @@ test.beforeEach(async ({ page }, testInfo) => {
 });
 ```
 
+Changing timeout for a fixture:
+
+```js js-flavor=js
+const { test: base, expect } = require('@playwright/test');
+
+const test = base.extend({
+  slowFixture: async ({}, use) => {
+    // Set a separate timeout for this fixture.
+    test.setTimeout(60000);
```

Comment:
Without looking at the docs, it seems like this would control the Test-level and not Fixture-level timeout. I think calling it `test.setFixtureTimeout(…)` would be more clear (or even `use.setFixtureTimeout` so it's not hanging off of `test` and is more coupled with the Fixture API).